server/routerlicious/packages/services-utils/src/scheduleJob.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+import * as cron from "node-cron";
+import { ILogger } from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
+
+export async function scheduleJob<T>(
```

Comment:
What if this returned a `removeScheduledJob` callback, instead of a task, which internally calls `task.stop; task.destroy()` if task is defined? If we move the feature flag logic out of this function, task should always be defined, but if it stays, it could be undefined, and you could handle that in the callback instead of relying on the caller to know that the task could be undefined or, more importantly, all the function calls needed to properly dispose of a task.

Reply:
This is a good point, I decided to go with your other idea of just keeping the feature flag logic outside of this function so task will always be defined - my plan is to call this function before initialising scriptorium lambda factory, and pass the task returned from this function to the lambda factory constructor, similar to how we pass mongo manager, so that the task can be cleaned up when dispose() is called